tests/unit/App.spec.js

```diff
@@ -85,11 +85,11 @@ Given("an App", () => {
       });
 
       Then("a request should be made", () => {
-        assert(spy.calledOnce, "spy is not being called 1 time");
+        expect(spy.calledOnce, "spy is not being called 1 time");
```

Comment:
This is not the right syntax for `expect`. Should be something like `expect(...).to.equal(...)`

Reply:
@jdesrosiers I've edited the syntax to be `expect(...).to.be.true`